src/Packages.res

```diff
@@ -538,9 +566,46 @@ let getStaticProps: Next.GetStaticProps.revalidate<props, unit> = async _ctx =>
       description: Belt.Option.getWithDefault(pkg["description"], ""),
       repositoryHref: Js.Null.fromOption(pkg["links"]["repository"]),
       npmHref: pkg["links"]["npm"],
+      searchScore: item["searchScore"],
+      maintenanceScore: item["score"]["detail"]["maintenance"],
+      score: item["score"],
     }
   })
 
+let getStaticProps: Next.GetStaticProps.revalidate<props, unit> = async _ctx => {
+  let baseUrl = "https://registry.npmjs.org/-/v1/search?text=keywords:rescript&size=250&maintenance=1.0&popularity=0.5&quality=0.9"
+
+  let (one, two, three) = await Js.Promise2.all3((
+    fetchNpmPackages(baseUrl),
+    fetchNpmPackages(baseUrl ++ "&from=250"),
+    fetchNpmPackages(baseUrl ++ "&from=500"),
+  ))
+
+  let (data1, data2, data3) = await Js.Promise2.all3((
+    one->Response.json,
+    two->Response.json,
+    three->Response.json,
+  ))
+
+  let unmaintained = []
+
+  let pkges =
+    parsePkgs(data1)
+    ->Js.Array2.concat(parsePkgs(data2))
+    ->Js.Array2.concat(parsePkgs(data3))
+    ->Js.Array2.filter(pkg => {
+      if [/* Allow list of names */]->Js.Array2.includes(pkg.name) {
```

Comment:
You could do a 

```res
let allowList = []
```

somehwere at the beginning of the file already then you don't need the comment here.

Reply:
Done.